webrev/src/main/java/org/openjdk/skara/webrev/ModifiedFileView.java

```diff
@@ -50,20 +50,25 @@ public ModifiedFileView(ReadOnlyRepository repo, Hash base, Hash head, List<Comm
         this.formatter = formatter;
         if (patch.isTextual()) {
             binaryContent = null;
-            oldContent = repo.lines(patch.source().path().get(), base).orElseThrow(() -> {
-                throw new IllegalArgumentException("Could not get content for file " +
-                                                   patch.source().path().get() +
-                                                   " at revision " + base);
-            });
+            var sourcePath = patch.source().path().orElseThrow(() ->
+                new IllegalArgumentException("Could not get source path for file with hash " +
+                                                   patch.source().hash() + " with target path" +
+                                                   patch.target().path().get())
```

Comment:
Is this `get()` safe though? Maybe you want to extract the target path into a variable as well, since the `patch.target().path().get()` is used a few times below as well? (It would reduce the number of `get()` calls`, so less scrutiny needed).

Reply:
Yeah, `path.target().path().get()` _should_ be safe. If the patch does not have `source` path nor have a `target` path, then something is seriously wrong :smile: